translations/ui-organizations/en.json

```diff
@@ -221,6 +221,7 @@
   "contactPeople.addContact": "Add contact",
   "contactPeople.newContact": "New",
   "contactPeople.noFindContactPlugin": "no find-contact plugin",
+  "contactPeople.remove": "Removed contact",
```

Comment:
maybe it's better to make key corresponding to value, contactPeople.removedContact. what do you think?

Reply:
oh yeah, I wanted to use this, mb ctrl + z